app/test_main.py

```diff
@@ -1,3 +1,63 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,expected",
+    [
+        pytest.param(
+            0,
+            0,
+            [0, 0],
+            id="cat and dog ages equal zero"
+        ),
+        pytest.param(
+            14,
+            14,
+            [0, 0],
+            id="cat and dog ages less then human"
+        ),
+        pytest.param(
+            15,
+            15,
+            [1, 1],
+            id="15 cat/dog years should convert into 1 human age."
+        ),
+        pytest.param(
+            23,
+            23,
+            [1, 1],
+            id="23 cat/dog years should convert into 1 human age."
+        ),
+        pytest.param(
+            24,
+            24,
+            [2, 2],
+            id="24 cat/dog years should convert into 2 human age."
+        ),
+        pytest.param(
+            27,
+            27,
+            [2, 2],
+            id="27 cat/dog years should convert into 2 human age."
+        ),
+        pytest.param(
+            28,
+            28,
+            [3, 2],
```

Comment:
The expected output for 28 cat/dog years should not be [3, 2] if the conversion logic is consistent with the other test cases. Please verify the expected output for this case.